### ISSUE TITLE
Attributes containing hyphens produces invalid generated code

### ISSUE THREAD

**Issue by mikegleasonjr** [tivac/mithril-objectify#35]
```
m('div', {
  'data-foo': 'bar'
}
```
generates

```
attrs: {
  data-foo: 'bar'
}
```

Which breaks...

EDIT: in the meantime the workaround is to do something like

```
var a = {};
a['data-foo'] = 'bar';
m('div', a);
```

**Comment by tivac** [comment]
Bleh, I'll get that fixed up soon. Sorry about that!

**Comment by tivac** [comment]
Fixed in `mithril-objectify@3.0.1`

**Comment by mikegleasonjr** [comment]
Nice shizzle!
